packages/components/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,263 @@
+    ...pressableBaseStyle,
+    borderWidth: 2,
+    borderRadius: spacing.vadsSpace2xs,
+    paddingTop: spacing.vadsSpaceSm,
+    paddingBottom: spacing.vadsSpaceSm,
+    paddingLeft: spacing.vadsSpaceSm,
+    paddingRight: spacing.vadsSpaceMd,
+    borderColor: checked
+      ? theme.vadsColorFormsBorderActive
+      : theme.vadsColorFormsBorderSubtle,
+    backgroundColor: checked
+      ? theme.vadsColorFormsSurfaceActive
+      : theme.vadsColorSurfaceDefault,
+  }
+
+  /**
+   * Icon
+   */
+
+  const iconViewStyle: ViewStyle = {
+    alignItems: 'center',
+    justifyContent: 'center',
+    paddingTop: 3,
+    paddingBottom: 3,
+  }
+
+  const _icon = (
+    <View style={iconViewStyle}>
+      <Icon
+        name={
+          indeterminate
+            ? 'IndeterminateCheckBox'
+            : checked
+              ? 'CheckBox'
+              : 'CheckBoxOutlineBlank'
+        }
+        fill={
+          checked
+            ? theme.vadsColorFormsForegroundActive
+            : theme.vadsColorFormsBorderDefault
+        }
+      />
+    </View>
+  )
+
+  /**
+   * Label
+   */
+
+  const labelProps = {
+    'aria-label': labelA11y || label,
+    style: {
+      ...baseTextStyle,
+      fontFamily: error ? 'SourceSansPro-Bold' : 'SourceSansPro-Regular',
+      fontSize: 20,
+      lineHeight: 30,
+    },
+  }
+
+  const requiredStyle = {
+    color: theme.vadsColorForegroundError,
+  }
+
+  const _label = (
+    <Text {...labelProps}>
+      {label}
+      {required && <Text style={requiredStyle}>{` (*${t('required')})`}</Text>}
+    </Text>
+  )
+
+  /**
+   * Header
+   */
+
+  const headerProps = {
+    'aria-label': headerA11y || header,
+    style: { ...baseTextStyle, fontSize: 20, lineHeight: 30 },
+  }
+
+  const _header = header && (
+    <>
+      <Text {...headerProps}>{header}</Text>
+      <Spacer size="xs" />
+    </>
+  )
+
+  /**
+   * Hint
+   */
+
+  const hintProps = {
+    'aria-label': hintA11y || hint,
+    style: {
+      ...baseTextStyle,
+      color: theme.vadsColorForegroundSubtle,
+      fontSize: 16,
+      lineHeight: 22,
+    },
+  }
+
+  const _hint = hint && (
+    <>
+      <Text {...hintProps}>{hint}</Text>
+      <Spacer size="xs" />
+    </>
+  )
+
+  /**
+   * Error
+   */
+
+  const errorProps = {
+    'aria-label': `${t('error')}: ${errorA11y || error}`,
+    style: {
+      fontFamily: 'SourceSansPro-Bold',
+      color: theme.vadsColorForegroundError,
+      fontSize: 16,
+      lineHeight: 22,
+    },
+  }
+
+  const _error = error && (
+    <>
+      <Text {...errorProps}>{error}</Text>
+      <Spacer size="xs" />
+    </>
+  )
+
+  /**
+   * Description
+   */
+
+  const descriptionProps = {
+    'aria-label': descriptionA11y || description,
+    style: { ...baseTextStyle, fontSize: 16, lineHeight: 22 },
+  }
+
+  const _description = description && (
+    <>
+      <Spacer size="xs" />
+      <Text {...descriptionProps}>{description}</Text>
+    </>
+  )
+
+  return (
```

Comment:
Minor and nitpicky, but noting that the the order above not matching the order here is a little disorienting (probably more from looking at it in browser without Cmd + clicking around).